plugins/modules/proxmox.py

```diff
@@ -415,11 +424,23 @@ def is_template_container(self, node, vmid):
         return config['template']
 
     def create_instance(self, vmid, node, disk, storage, cpus, memory, swap, timeout, clone, **kwargs):
+        # Available only in PVE 7
+        only_v7 = ['tags']
+
         proxmox_node = self.proxmox_api.nodes(node)
 
         # Remove all empty kwarg entries
         kwargs = dict((k, v) for k, v in kwargs.items() if v is not None)
 
+        version = self.version()
+        pve_major_version = 3 if version < LooseVersion('4.0') else version.version[0]
+
+        # The features work only on PVE 7
+        if pve_major_version < 7:
+            for p in only_v7:
```

Comment:
Then this would become:
```suggestion
        for option, version in minimum_version.items():
            if pve_major_version < version:
```
(with `p` replaced by `option` below)